homeworks/magurmarian/hw02/job.py

```diff
@@ -0,0 +1,28 @@
+"""There are two numbers. Each of them can be negative, floating etc.
+You need to create a job which will accept two numbers as the arguments.
+This job has to run in parallel 4 different tests: multiplication, division, addition, subtraction.
+Each of these tests will perform an appropriate math operation on given numbers.
+A test has to fail if the calculation result is less then 0, otherwise, the test is passed."""
+import argparse
+import sys
+from pyats.easypy import Task
+
+
+def new_task(runtime, test, num1, num2):
+    return Task(testscript=test, runtime=runtime, num1=num1, num2=num2)
+
+
+parser = argparse.ArgumentParser(description="math calc")
```

Comment:
I would move this part of code below the functions for better readability.

Reply:
Done!